generators/app/templates/src/app/screens/Login/index.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import Login from './index';
+
+describe('#Login', () => {
```

Comment:
Could we maybe abstract this to a generic test? Seems like this in particular would work with any component.  It might be nice to create  these generic test cases! (Do we have a card for this? :thinking: )

Reply:
Mmm interesting... I'm not sure, normally the tests won't be this simple. I just wanted to leave a snapshot testing example. This should be more complex depending on how the login works. Not sure that can be extracted, if I eventually want to add more tests in each screen.